Samples/SVGViewer/SvgViewer.cs

```diff
@@ -22,12 +22,20 @@ private void Open_Click(object sender, EventArgs e)
                     var svgDoc = SvgDocument.Open(openSvgFile.FileName);
                     RenderSvg(svgDoc);
 
-                    var xmlDoc = new XmlDocument
+                    textBox1.TextChanged -= new EventHandler(TextBox1_TextChanged);

```

Comment:
I don't understand this - how do you unsubscribe from a handler you just created?
Can you please elaborate a bit what this PR does?

Reply:
@mrbean-bremen 

Thank you for your review.

This event handler has already been registered in Designer.
Update text on line 35 of updated file.
Temporarily unsubscribe TextChanged event handler so that SVG is not loaded again by TextChanged event.

Reply:
Ah, thanks - now it makes sense to me!